Source/Utils/Average.H

```diff
@@ -89,6 +89,55 @@ namespace Average{
                         const int ngrow,
                         const int scomp=0,
                         const int ncomp=1 );
+
+    AMREX_GPU_HOST_DEVICE
+    AMREX_FORCE_INLINE
+    Real FineToCoarse ( Array4<Real const> const& mf_fp_arr,
```

Comment:
The name `FineToCoarse` is misleading, because in the rest of the code it refers to interpolation from the fine patch to the coarse patch of the same MR level, while this function simply interpolates from 1 resolution to another. Could it be another name? `Coarsen` for instance.